scripts/initialize.js

```diff
@@ -19,22 +19,9 @@
 	};
 
 	AblePlayer.prototype.getRootPath = function() {
-
-		// returns Able Player root path (assumes ableplayer.js is in /build, one directory removed from root)
-		var scripts, i, scriptSrc, scriptFile, fullPath, ablePath, parentFolderIndex, rootPath;
-		scripts= document.getElementsByTagName('script');
-		for (i=0; i < scripts.length; i++) {
-			scriptSrc = scripts[i].src;
-			scriptFile = scriptSrc.substr(scriptSrc.lastIndexOf('/'));
-			if (scriptFile.indexOf('ableplayer') !== -1) {
-				// this is the ableplayerscript
-				fullPath = scriptSrc.split('?')[0]; // remove any ? params
-				break;
-			}
-		}
-		ablePath= fullPath.split('/').slice(0, -1).join('/'); // remove last filename part of path
-		parentFolderIndex = ablePath.lastIndexOf('/');
-		rootPath = ablePath.substring(0, parentFolderIndex) + '/';
+		// no need to find root path from script tag anymore, just
+		// return 'ableplayer' (most probably won't need this anyway)
+		rootPath = "ableplayer/";
```

Comment:
Maybe remove the whole `getRootPath` function and the code that uses this.

Reply:
Or add an `TODO` comment to remove the code later

Reply:
Is there still code that uses this? If there is, I'd say there is a high chance that it doesn't work anymore.

Reply:
Yeh there is and they probably don't -  I was also procrastinating on that knowing that it means I have to spend more time digging into what else doesn't work... when currently it when I test it it works fine 😁 I'll now have a look and see what else doesn't work..

Reply:
Ok fixed! Turns out there's only one other place that I missed 😅